www/tilesource-iris.html

```diff
@@ -0,0 +1,166 @@
+<h2>
+    Example: Iris TileSource Support
+</h2>
+<p>
+    The <strong>IrisTileSource</strong> is an OpenSeadragon tilesource implementation for viewing images served by an Iris-compatible tile server. It is designed to efficiently display large, high-resolution images by loading only the visible tiles at the appropriate zoom level.
```

Comment:
All OSD tile sources load only the visible tiles at the appropriate zoom level, so it's not a distinguishing feature of Iris. Instead of saying that, perhaps talk more about what makes Iris different.